samples/Localization.StarterWeb/Localization.StarterWeb.csproj

```diff
@@ -42,7 +34,6 @@
 
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.2.0-*" />
-    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.1-*" />
```

Comment:
This tool is used to set secrets from command line. Even if not in automated tests, it's still used during "dotnet restore". This is one of the "validations" this project provides. Restore would fail if broke the secretmanager tool.